repository/model/settings.go

```diff
@@ -0,0 +1,92 @@
+package model
+
+import (
+	"encoding/json"
+	"fmt"
+
+	svcmodel "release-manager/service/model"
+)
+
+const (
+	keyOrganizationName      = "organization_name"
+	keyDefaultReleaseMessage = "default_release_message"
+	keySlack                 = "slack"
+	keyGithub                = "github"
+)
+
+type Settings struct {
+	OrganizationName      string
+	DefaultReleaseMessage string
+	Slack                 IntegrationSettings
+	Github                IntegrationSettings
+}
+
+type IntegrationSettings struct {
+	Enabled bool   `json:"enabled"`
+	Token   string `json:"token"`
+}
+
+func (c *Settings) MarshalJSON() ([]byte, error) {
```

Comment:
Why `c`?

Reply:
And it's not used anywhere.

Reply:
```
	err := r.client.
		DB.From(r.entity).
		Upsert(&data).
		ExecuteWithContext(ctx, nil)
```

When struct is passed to supabase client, client calls json.Marshal() and i need a custom marshal function to convert struct to key value pairs to save them in db.